mono/metadata/object-internals.h

```diff
@@ -705,8 +705,11 @@ mono_raise_exception_deprecated (MonoException *ex);
 void
 mono_reraise_exception_deprecated (MonoException *ex);
 
-void
-mono_raise_exception_with_context (MonoException *ex, MonoContext *ctx);
+void // FIXMEcoop remove this
+mono_raise_exception_with_context (MonoException* ex, MonoContext *ctx);
+
+void // FIXMEcoop rename this
```

Comment:
Line damage; we know any `MonoObject*` needs to be removed, commenting doesn't bring any more explanation to why it needs to be done.

Reply:
Sure.